app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -202,18 +197,11 @@ public void addOrUpdateAlert(final Alert alert, final Organization organization)
             }
 
             ebeanAlert.setOrganization(models.ebean.Organization.findByOrganization(organization));
-            ebeanAlert.setCluster(alert.getCluster());
             ebeanAlert.setUuid(alert.getId());
-            ebeanAlert.setMetric(alert.getMetric());
-            ebeanAlert.setContext(alert.getContext());
-            ebeanAlert.setNagiosExtension(convertToEbeanNagiosExtension(alert.getNagiosExtension()));
             ebeanAlert.setName(alert.getName());
-            ebeanAlert.setOperator(alert.getOperator());
-            ebeanAlert.setPeriod(alert.getPeriod().toStandardSeconds().getSeconds());
-            ebeanAlert.setQuantityValue(alert.getValue().getValue());
-            ebeanAlert.setQuantityUnit(alert.getValue().getUnit().orElse(null));
-            ebeanAlert.setStatistic(alert.getStatistic());
-            ebeanAlert.setService(alert.getService());
+            ebeanAlert.setQuery(alert.getQuery());
+            ebeanAlert.setPeriod(alert.getCheckInterval().toStandardSeconds().getSeconds());
+            ebeanAlert.setComment(alert.getComment());
```

Comment:
Should we also have enabled/disabled? Probably also with a deadline (e.g. snooze). This can be an issue.